src/patternfly/components/Content/examples/Content.md

```diff
@@ -116,4 +119,5 @@ This component is an exception to the variable system since we style type select
 ### Usage
 | Class | Applied to | Outcome |
 | -- | -- | -- |
-| `.pf-c-content` | `<div>`, `<section>`, or `<article>` | Generates vertical rythm and typographic treatment to html elements |
+| `.pf-c-content` | `<div>`, `<section>`, or `<article>` | Generates vertical rythm and typographic treatment to html elements. |
+| `.pf-m-visited-link` | `.pf-c-content`, `<a>` | Modifies a visited link styling to `--pf-color-purple-600`. |
```

Comment:
fixed the class name and also made the description less specific - if we update the color, we won't have to update this description.

```suggestion
| `.pf-m-visited` | `.pf-c-content`, `<a>` | Modifies all links in a content block to include visited styles. Can also be applied to a single link in a content block. |
```